rotkehlchen/tasks/calendar.py

```diff
@@ -410,6 +422,60 @@ def maybe_create_airdrop_claim_reminder(self) -> None:
             error_msg='Failed to add the airdrop claim reminders',
         )
 
+    def maybe_create_l2_bridging_reminder(self) -> None:
+        """Creates calendar reminders for L2 bridge deposit claims.
+        Tracks deposits on Base, Optimism, and Arbitrum networks.
+        """
+        bridge_events: list[EvmEvent] = []
+        seven_days_in_ms = ts_sec_to_ms(Timestamp(WEEK_IN_SECONDS))
+        locations_to_counterparties = [
+            (Location.BASE, CPT_BASE),
+            (Location.OPTIMISM, CPT_OPTIMISM),
+            (Location.ARBITRUM_ONE, CPT_ARBITRUM_ONE),
+        ]
+        db_history_events = DBHistoryEvents(database=self.database)
+        with self.database.conn.read_ctx() as cursor:
+            for location, counterparty in locations_to_counterparties:
+                bridge_events.extend(db_history_events.get_history_events(
+                    cursor=cursor,
+                    has_premium=True,
+                    filter_query=EvmEventFilterQuery.make(
+                        and_op=True,
+                        location=location,
+                        counterparties=[counterparty],
+                        event_types=[HistoryEventType.DEPOSIT],
+                        event_subtypes=[HistoryEventSubType.BRIDGE],
+                    ),
+                ))
+
+        now = ts_now_in_ms()
+        bridge_calendar_entries: list[int] = []
+        for bridge_event in bridge_events:
+            if now - bridge_event.timestamp > seven_days_in_ms:
+                continue
+
+            try:
+                entry_id = self.create_or_update_calendar_entry_from_event(
+                    name=f'Claim {bridge_event.balance.amount} {bridge_event.asset.symbol_or_name()} bridge deposit on Ethereum',  # noqa: E501
+                    event=bridge_event,
+                    color=BRIDGE_CALENDAR_COLOR,
+                    counterparty=bridge_event.counterparty,  # type: ignore[arg-type]  # counterparty is always present
+                    timestamp=ts_ms_to_sec(TimestampMS(bridge_event.timestamp + WEEK_IN_SECONDS)),
+                    description=f'Bridge deposit of {bridge_event.balance.amount} {bridge_event.asset.symbol_or_name()} is ready to claim on Ethereum',  # noqa: E501
+                )
+
+                if entry_id is not None:
+                    bridge_calendar_entries.append(entry_id)
```

Comment:
use the walrus operator here

```suggestion
                if (entry_id := self.create_or_update_calendar_entry_from_event(
                    name=f'Claim {bridge_event.balance.amount} {bridge_event.asset.symbol_or_name()} bridge deposit on Ethereum',  # noqa: E501
                    event=bridge_event,
                    color=BRIDGE_CALENDAR_COLOR,
                    counterparty=bridge_event.counterparty,  # type: ignore[arg-type]  # counterparty is always present
                    timestamp=ts_ms_to_sec(TimestampMS(bridge_event.timestamp + WEEK_IN_SECONDS)),
                    description=f'Bridge deposit of {bridge_event.balance.amount} {bridge_event.asset.symbol_or_name()} is ready to claim on Ethereum',  # noqa: E501
                )) is not None:
                    bridge_calendar_entries.append(entry_id)
```